test/FunctionalTests/Microsoft.OData.Core.Tests/ScenarioTests/Roundtrip/JsonReaderWriterInjectionTests.cs

```diff
@@ -139,6 +322,39 @@ private static ODataResource GetReadedResource(string messageContent, EdmModel m
             }
         }
 
+        private static IList<ODataResource> GetReadedResourceWithNestedInfo(string messageContent, EdmModel model, IEdmEntitySetBase entitySet, EdmEntityType entityType, IServiceProvider container)

```

Comment:
>GetReadedResourceWithNestedInfo [](start = 44, length = 31)

where's the writing test cases. 

for example: I have a $select=HomeAddress/Street

I'd have test case to verify the output context uri is correct.

And, for $select=HomeAddress($select=Street) can output the same context uri.

Reply:
Write test are already present
#Resolved